osFiles/Simulators/iOS/20A360.json

```diff
@@ -0,0 +1,25 @@
+{
+    "osStr": "iOS",
+    "version": "16 Simulator Runtime",
```

Comment:
inconsistent naming

Reply:
which way makes more sense to you? I'm leaning towards "Simulator" but can go either way

Reply:
agreed

Reply:
Fixed and pushed